tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,79 @@
+"""Task 1 and 2 of Module 2."""
+
+from collections import defaultdict
+
+LIGHT_WEIGHT_THRESHOLD = 60
+HEAVY_WEIGHT_THRESHOLD = 70
+MAX_INVENTORY_CAPACITY = 80
+UNWANTED_ITEMS = ['rubbish', 'chewed gum', 'used tissue']
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+
+dragon_loot = [
+    'gold coin', 'chewed gum', 'dagger', 'gold coin', 'gold coin', 'ruby',
+    'rubbish', 'chewed gum', 'used tissue']
+
+
+def display_inventory(inventory):
+    """Check the inventory and provides weight/speed dependency status."""
+    item_total = 0
+
+    print("Inventory:")
+
+    for item_name, item_count in inventory.items():
+
```

Comment:
I would rather keep it compact and remove this empty line. And maybe empty line after print.